en/tools/config-catalog-generator/data/configs.json

```diff
@@ -1632,6 +1632,76 @@
+                            "possible": "",
+                            "description": "The email address you use to send emails."
+                        },
+                        {
+                            "name": "username",
+                            "type": "string",
+                            "required": true,
+                            "default": "",
+                            "possible": "",
+                            "description": "The email address used to authenticate the mail server. This can be the same email address as the from_address."
+                        },
+                        {
+                            "name": "password",
+                            "type": "string",
+                            "required": true,
+                            "default": "",
+                            "possible": "",
+                            "description": "Password used to authenticate the mail server."
+                        },
+                        {
+                            "name": "hostname",
+                            "type": "string",
+                            "required": false,
+                            "default": "",
+                            "possible": "",
+                            "description": "The SMTP server to connect to."
+                        },
+                        {
+                            "name": "port",
+                            "type": "string",
+                            "required": false,
+                            "default": "25",
+                            "possible": "",
+                            "description": "The SMTP server port to connect to, if the connect() method does not explicitly specify one. Defaults to 25."
+                        },
+                        {
+                            "name": "enable_start_tls",
+                            "type": "string",
+                            "required": false,
+                            "default": false,
+                            "possible": "",
+                            "description": "If true, enables the use of the `STARTTLS` command (if supported by the before issuing any login commands. Note that an appropriate trust store must configured so that the client will trust the certificate of the server. Defaults to false."
+                        },
+                        {
+                            "name": "enable_authentication",
+                            "type": "string",
+                            "required": false,
+                            "default": false,
+                            "possible": "",
+                            "description": "If true, it attempts to authenticate the user using the AUTH command. Defaults to false."
+                        }
+                        
+                    ]
+                }
+            ],
+            "exampleFile": "apim.notification.toml"
         }
     ]
 }
```

Comment:
Add newline at the end of file

Reply:
Extra line was not added since this file is only used to automatically generate config-catalog.md